src/cli.rs

```diff
@@ -117,6 +117,19 @@ pub struct RootOpts {
     )]
     pub config_paths_yaml: Vec<PathBuf>,
 
+    /// Read pipeline configuration from files in one or more directories.
+    /// File format is detected from the file name.
+    ///
+    /// Files not ending in .toml, .json, .yaml, or .yml will be ignored.
```

Comment:
Did we determine this was the actual behavior?

Reply:
considering it has to behave like the config files, yes.